chainer/datasets/cifar.py

```diff
@@ -106,69 +152,8 @@ def _preprocess_cifar(images, labels, withlabel, ndim, scale):
         return images
 
 
-def _retrieve_cifar_100():
-    root = download.get_dataset_directory('pfnet/chainer/cifar')
-    path = os.path.join(root, 'cifar-100.npz')
-    url = 'https://www.cs.toronto.edu/~kriz/cifar-100-python.tar.gz'
-
-    def creator(path):
-
-        def load(archive, file_name):
-            d = _pickle_load(archive.extractfile(file_name))
-            x = d['data'].reshape((-1, 3072))
-            y = numpy.array(d['fine_labels'], dtype=numpy.uint8)
-            return x, y
-
-        archive_path = download.cached_download(url)
-        with tarfile.open(archive_path, 'r:gz') as archive:
-            train_x, train_y = load(archive, 'cifar-100-python/train')
-            test_x, test_y = load(archive, 'cifar-100-python/test')
-
-        numpy.savez_compressed(path, train_x=train_x, train_y=train_y,
-                               test_x=test_x, test_y=test_y)
-        return {'train_x': train_x, 'train_y': train_y,
-                'test_x': test_x, 'test_y': test_y}
-
-    return download.cache_or_load_file(path, creator, numpy.load)
-
-
-def _retrieve_cifar(name):
-    root = download.get_dataset_directory('pfnet/chainer/cifar')
-    path = os.path.join(root, '{}.npz'.format(name))
-    url = 'https://www.cs.toronto.edu/~kriz/{}-python.tar.gz'.format(name)
-
-    def creator(path):
-        archive_path = download.cached_download(url)
-
-        train_x = numpy.empty((5, 10000, 3072), dtype=numpy.uint8)
-        train_y = numpy.empty((5, 10000), dtype=numpy.uint8)
-        test_y = numpy.empty(10000, dtype=numpy.uint8)
-
-        dir_name = '{}-batches-py'.format(name)
-
-        with tarfile.open(archive_path, 'r:gz') as archive:
-            # training set
-            for i in range(5):
-                file_name = '{}/data_batch_{}'.format(dir_name, i + 1)
-                d = _pickle_load(archive.extractfile(file_name))
-                train_x[i] = d['data']
-                train_y[i] = d['labels']
-
-            # test set
-            file_name = '{}/test_batch'.format(dir_name)
-            d = _pickle_load(archive.extractfile(file_name))
-            test_x = d['data']
-            test_y[...] = d['labels']  # copy to array
-
-        train_x = train_x.reshape(50000, 3072)
-        train_y = train_y.reshape(50000)
-
-        numpy.savez_compressed(path, train_x=train_x, train_y=train_y,
-                               test_x=test_x, test_y=test_y)
-        return {'train_x': train_x, 'train_y': train_y,
-                'test_x': test_x, 'test_y': test_y}
-
-    return download.cache_or_load_file(path, creator, numpy.load)
+_root = download.get_dataset_directory(os.path.join('pfnet', 'chainer',
```

Comment:
Do we need to call this method here? I feel it is enough to call it in `_get_cifar`.